pkg/skaffold/debug/transform.go

```diff
@@ -122,16 +125,42 @@ func transformPodSpec(metadata *metav1.ObjectMeta, podSpec *v1.PodSpec, retrieve
 	}
 	// containers are required to have unique name within a pod
 	configurations := make(map[string]map[string]interface{})
+	requiredSupportImages := make(map[string]string)
+	var supportFilesRequired []*v1.Container
 	for i := range podSpec.Containers {
 		container := &podSpec.Containers[i]
 		// we only reconfigure build artifacts
```

Comment:
Good point.  The `retrieveImageConfiguration` function passed in from debug.go returns an error for a container that is not a built artifact.  I'll reword this and rework the `transformContainer` so that the `imageConfiguration` is obtained here.